.github/workflows/screener-run.yml

```diff
@@ -19,7 +19,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{github.event.workflow_run.head_branch}}
```

Comment:
why is this being removed ?

Reply:
Found out that checking out user code is bad practice when using the `workflow_run` trigger event, and after the merge the workflows will run perfectly well without the need to check out to other branches.